src/traversals.js

```diff
@@ -1,7 +1,8 @@
 import { Tree } from './util/tree'
 
 export let markForUpdate = Tree.walk(x => {
-  x.markedForUpdate = true
+  if (x.paused) x.missedUpdate = true
+  else x.markedForUpdate = true
```

Comment:
This is probably worse, but fyi
`x[x.paused ? 'missedUpdate' : 'markedForUpdate'] = true`